pkgs/by-name/in/intel-graphics-compiler/package.nix

```diff
@@ -22,24 +22,24 @@ let
   vc_intrinsics_src = fetchFromGitHub {
     owner = "intel";
     repo = "vc-intrinsics";
-    rev = "v0.19.0";
-    hash = "sha256-vOK7xfOR+aDpdGd8oOFLJc1Ct1S5BCJmLN6Ubn5wlkQ=";
+    rev = "v0.22.1";
+    hash = "sha256-dSK+kNEZoF4bBx24S0No9aZLZiHK0U9TR1jRyEBL+2U=";
   };
 
   inherit (llvmPackages_14) lld llvm;
   inherit (if buildWithPatches then opencl-clang else llvmPackages_14) clang libclang;
   spirv-llvm-translator' = spirv-llvm-translator.override { inherit llvm; };
 in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "intel-graphics-compiler";
-  version = "1.0.17384.11";
+  version = "2.9.2";
```

Comment:
We also want to stay on 2.8.3 https://github.com/intel/intel-graphics-compiler/releases/tag/v2.8.3